infrastructure/smart-contracts/test/tinykeys/RefereeBulkSubmissions.mjs

```diff
@@ -318,7 +326,15 @@ function BulkSubmissionPermissions(deployInfrastructure) {
         const poolAddress = await createPool(poolFactory, addr1, winningKeys);
 
         // Stake the remaining keys in the pool minus the one that was submitted at pool creation
-        await poolFactory.connect(addr1).stakeKeys(poolAddress, addr1MintedKeyIds.slice(1));
+        const batchSize = 100;
+        for (let i = 2; i < keysToMintForAddr1; i += batchSize) {
+            let rangeEnd = i + batchSize;
+            if (i + batchSize > keysToMintForAddr1) {
+                rangeEnd = keysToMintForAddr1;
+            }
+            let keyIdsToStake = addr1MintedKeyIds.slice(i, rangeEnd);
+            await poolFactory.connect(addr1).stakeKeys(poolAddress, keyIdsToStake);
+        }
```

Comment:
Any thoughts on creating a helper function in utils to handle bulk staking?

Since we need to do this in virtually every test case now, would it make sense to abstract this?

Reply:
Yes thats a good idea, but it seems out of scope for this story, we should add a ticket to refactor this please.